docs/book/src/getting-started/installation.md

```diff
@@ -88,3 +88,4 @@ Now that the Secrets Store CSI Driver has been deployed, select a provider from
 - [Azure Provider](https://azure.github.io/secrets-store-csi-driver-provider-azure/)
 - [GCP Provider](https://github.com/GoogleCloudPlatform/secrets-store-csi-driver-provider-gcp)
 - [Vault Provider](https://github.com/hashicorp/secrets-store-csi-driver-provider-vault)
+- [Akeyless Provider](https://github.com/akeylesslabs/akeyless-csi-provider)
```

Comment:
The current list of providers is alphabetical, so move this to the top of the list.

Reply:
Done in latest commit.